addons/mail/models/mail_message.py

```diff
@@ -848,26 +848,27 @@ def toggle_message_starred(self):
     @api.model
     def _message_fetch(self, domain, search_term=None, before=None, after=None, around=None, limit=30):
         res = {}
+        domain = Domain(domain)
```

Comment:
I suppose changing all callers of this method to pass an actual Domain as param was considered out of scope of the current PR?

Reply:
Yes, sometimes there are a lot of changes and I don't want to find them all.